cloudlift/deployment/service_template_generator.py

```diff
@@ -959,6 +981,71 @@ def _get_desired_task_count_for_service(self, service_name):
             return self.desired_counts[service_name]
         else:
             return 0
+    def _firelens_container_def_args_override(self, configuration, container_def_args):
+        if configuration.get('logging') == 'awsfirelens':
+            container_def_args['DependsOn'] = container_def_args.get('DependsOn', []) + [ContainerDependency(ContainerName='fluentbit-firelens-sidecar', Condition='START')]
```

Comment:
Lets add a condition to check if sidecar container exists, also check if condition will be "START" or "healthy" @rokibulislaam

Reply:
done